### ISSUE TITLE
Not using oredictionary for the chest in bag recipe

### ISSUE THREAD

**Issue by Zero9o1o** [lorddusk/Bagginses#80]
### What were you doing when the issue happened? ###
Trying to craft a bag using quark versions of wooden chests.

### What did you expect to happen? ###
That the recipe for bags would use the oredictionary ore:chestWood so the bag could be crafted with any wooden chest.

### What happened instead? ###
The bag won't craft due to needing only vanilla chest

### Crashlog on pastebin/gist ###
No crashlog but ref to code line for the crafting recipe.
[Recipe](https://github.com/lorddusk/Bagginses/blob/4c50900c43f546b16d0fdae69b74181c752e03a0/src/main/java/nl/lang2619/bagginses/items/ModItems.java#L110)

### Forge and Minecraft version? ###
Minecraft version: 1.10.2
Forge Version: 12.18.2.2124

**Comment by llloyd4** [comment]
Yeah, it only works with oak chests, not birch or any other species of wood. >.<
